server/helpers/JWTHelper.js

```
@@ -0,0 +1,44 @@
+import jsonWebToken from 'jsonwebtoken';
+import { config } from 'dotenv';
+import { secretKey } from '../config/config';
+
+config();
+
+const expirationTime = 86400;
+
+/**
+ * @description This class is for JWT token generation and verification
+ */
+class JWTHelper {
+  /**
+   * @description This function generates JWT tokens
+   * @param {object} userObject
+   * @returns {string} token
+   */
+  static generateToken(userObject){
+    if(!userObject || !userObject.firstName) {
+      return 'Please supply a valid user object.';
```

Nice work @davidshare. But we could throw an error here